toolkit/scripts/pkggen.mk

```
@@ -110,7 +120,19 @@ $(graph_file): $(specs_file) $(go-grapher) $(toolchain_rpms)
 		$(if $(filter y,$(ENABLE_MEM_PROFILE)),--enable-mem-prof) \
 		$(if $(filter y,$(ENABLE_TRACE)),--enable-trace) \
 		--timestamp-file=$(TIMESTAMP_DIR)/grapher.jsonl \
-		--output $@
+		--output $@ \
+		$(if $(filter y,$(RESOLVE_CYCLES_FROM_UPSTREAM)), --resolve-cycles-from-upstream) \
+		$(if $(filter y,$(USE_PREVIEW_REPO)), --use-preview-repo) \
```

We added this recently.
```suggestion
		$(if $(filter y,$(USE_PREVIEW_REPO)), --use-preview-repo) \
        $(if $(filter y,$(DISABLE_DEFAULT_REPOS)), --disable-default-repos) \
```